### PR TITLE
chore: fixed typo, instance data structure and readme

### PR DESCRIPTION
**Pull request by BRGustavoRibeiro**
- The 'official' tag has been renamed to 'verified', as it can help official support to long-term, serious and well-established instances in the future.
- The 'display' tag has been added in case an instance admin wants to hide itself from the website without needing to remove it from the list (for temporary reasons, like an instance overload or extremely high usage, for example. Lemmy suffered from this problem a few weeks ago).

### CODE REVIEW

**Review by MaddyUnderStars — APPROVED** [review]

File: instances.json [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-	}
+  {
+    "name": "Spacebar",
+    "url": "https://spacebar.chat",
+    "description": "The official Spacebar instance.",
+    "image": "https://raw.githubusercontent.com/spacebarchat/spacebarchat/master/branding/png/Spacebar__Icon-Discord.png",
+    "display": true,
+    "verified": true,
+    "country": "br",
+    "language": "pt"
```
Comment (MaddyUnderStars) [inline instances.json:10]: brazil? portuguese? the official instance is located in the the EU and is english-only

**Review by MaddyUnderStars — CHANGES_REQUESTED** [review]

File: instances.json (hunk 1 above)
Comment (BRGustavoRibeiro) [reply instances.json:10]: Sorry, I confused myself

This was written when I was tired af and coding almost asleep 🫠
Will fix

File: instances.json [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+  },
+  {
+    "name": "Fastbar",
+    "url": "https://spacebar.greysilly7.xyz",
+    "description": "The best Spacebar instance with 95% uptime, running under on a NVME drive running with bleeding edge stuff <3",
+    "image": "https://github.com/greysilly7.png",
+    "display": true,
+    "verified": false,
+    "country": "xx",
+    "language": "xx"
```
[the remark this answers is not in the document]
Comment (BRGustavoRibeiro) [reply instances.json:20]: I was probably following the [ISO 3166-1 alpha-2, where XX](
https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#XX) is used by WIPO to design unknown states, entities or organizations.

But I agree that _undefined_ would be a better fit. Will fix

**Comment by MaddyUnderStars** [comment]
thank you!

**Pull request merged**